packages/@react-native-community/changelog-generator/src/changelog-generator.ts

```diff
@@ -184,15 +181,15 @@ function filterRevertCommits(commits: Commit[]) {
       return true;
     })
     .filter(item => {
-      let text = item.commit.message.split("\n")[0].toLowerCase();
-      revertCommits.forEach(revertCommit => {
+      const text = item.commit.message.split("\n")[0].toLowerCase();
+      for (const revertCommit of revertCommits) {
```

Comment:
We were previously attmempting true/false to the outer filter function, but were instead just returning to `forEach` which didn't use them.